test/tests/httptests.ts

```diff
@@ -43,7 +43,7 @@ describe('Http Tests', function () {
         assert(res.message.statusCode == 200, "status code should be 200");
         let body: string = await res.readBody();      
         let obj: any = JSON.parse(body);
-        assert(obj.url === "https://httpbin.org/get");
+        assert(obj.url === "http://httpbin.org/get");
```

Comment:
Any reason for these test changes?

Reply:
@damccorm For some reason, the tests were failing! Assertions on `obj.url` were not fulfilled as expected.  You could check/see this in the first commit where the builds failed as tests failed. I wonder why in the first place tests were the `https` module is not used, is asserting against urls with `https://`. But still I'm confused how they working fine previously and out-of-sudden first when opening this PR

Reply:
Hm, it looks like the payload just changed http://httpbin.org/get

Thanks for clarifying, this LGTM!